backend/handlers/handlers.go

```diff
@@ -686,6 +687,55 @@ func (handlers *Handlers) postDisconnectBaseHandler(r *http.Request) (interface{
 	return map[string]interface{}{"success": success}, nil
 }
 
+func (handlers *Handlers) postConnectElectrumHandler(r *http.Request) (interface{}, error) {
+	jsonBody := map[string]string{}
+	if err := json.NewDecoder(r.Body).Decode(&jsonBody); err != nil {
+		return nil, errp.WithStack(err)
+	}
+	bitboxBaseID := jsonBody["bitboxBaseID"]
+	bitboxBase := handlers.backend.BitBoxBasesRegistered()[bitboxBaseID]
+	bitboxBaseIP := bitboxBase.GetIP()
+	// 51002 is the bitbox bases's testnet nginx electrum ssl port
+	electrumIP := bitboxBaseIP + ":" + bitboxBase.GetElectrsRPCPort()
+
+	electrumCert, err := handlers.backend.DownloadCert(electrumIP)
+	if err != nil {
+		handlers.log.WithField("ElectrumIP: ", electrumIP).Error(err.Error())
+		return map[string]interface{}{
+			"success":      false,
+			"errorMessage": err.Error(),
+		}, nil
+	}
+
+	if err := handlers.backend.CheckElectrumServer(
+		electrumIP,
+		electrumCert); err != nil {
+		handlers.log.WithField("ElectrumIP: ", electrumIP).Error(err.Error())
+		return map[string]interface{}{
+			"success":      false,
+			"errorMessage": err.Error(),
+		}, nil
+	}
+
+	handlers.log.WithField("ElectrumIP:", electrumIP).Debug("Connecting to new electrum Server...")
+
+	config := handlers.backend.Config()
+	// BaseBtcConfig sets the TBTC configs to the provided cert and ip.
+	config.BaseBtcConfig(electrumIP, electrumCert, bitboxBase.GetNetwork())
+	// Disable Litecoin and Ethereum accounts - we do not want any more traffic hitting other servers
```

Comment:
I disagree. If I use the base for btc, it does not mean I only use btc ;) I'd remove this.

Reply:
I would not want my wallet app talking to any shift operated servers any more once I configure the base.

Reply:
But users don't really have the option unless we also provide them with a Litecoin and Ethereum node, or they would need to use a separate app

Reply:
I see the point, but at least there should be a message after it saying that those were disabled for that reason, and the user can re-enable them in the settings.

Reply:
This seems like a difficult UX decision, and I think we should provide as much transparency for the user as possible. 

My preference is to display a message and ask whether to disable backends for other chains (default: no).

Reason: if I'm a Bitcoin Maximalist (non-toxic) I don't use the other chains anyway. If I have Ethereum / Litecoin enabled, I'd probably want to keep the wallet functionality.

Reply:
Is it fine if we add proper notification flow in a separate pull request?

Reply:
sure